net-analyzer/nload/nload-0.7.4-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="Real time network traffic and bandwidth usage monitor for the text console"
```

Comment:
I think this is a bad translation and suggest
 DESCRIPTION="Real time network traffic monitor for the command line interface"

Reply:
OK, I'll fix the description and update the PR.